app/build.gradle

```diff
@@ -47,4 +47,10 @@ dependencies {
     implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
     implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
 
+    // QR Code Scanner by budiyev
+    implementation 'com.budiyev.android:code-scanner:2.1.0'
+
+    // For Handling permissions
+    implementation 'pub.devrel:easypermissions:3.0.0'
```

Comment:
Do we really need to use a library just to handle permissions?
@antailbaxt3r what do you think?

Reply:
Yes, use EasyPermissions